repo/modules.go

```diff
@@ -62,6 +66,9 @@ func importRepoRulesModules(filename string, _ *RemoteCache) (repos []Repo, err
 			continue
 		}
 		pathToModule[mod.Path] = mod
+		if mod.Replace != nil {
```

Comment:
We probably don't need `replacePathToModule`. Whenever the `replace` directive is set, it should be ok to use `mod.Replace.Path` as the key for `pathToModule`.

Reply:
:+1: